specification/deviceregistry/DeviceRegistry.Management/properties/ns_discoveredAssetsProperties.tsp

```diff
@@ -155,13 +175,43 @@ model NamespaceDiscoveredDatasetDataPoint is NamespaceDiscoveredDataPoint {
 @added(Versions.v2025_07_01_preview)
 model NamespaceDiscoveredEventDataPoint is NamespaceDiscoveredDataPoint;
 
+@doc("Defines the discovered event group properties.")
+@added(Versions.v2025_10_01)
+model NamespaceDiscoveredEventGroup {
```

Comment:
I might not see it, but arent the additions missing

@doc("UTC timestamp indicating when the xxxx was added or modified.")
  lastUpdatedOn?: utcDateTime;
  
  Not that I know much about how Akri or consumers of the information are supposed to use this field